samples/react/authentication-auth-code-flow/src/App.js

```diff
@@ -0,0 +1,100 @@
+import { useState, useEffect, useMemo } from 'react';
+import { CogniteClient } from '@cognite/sdk';
+
+const AAD_CONFIG = {
+  clientId: "AAD-APP-ID",
+  openIdConfigurationUrl: 'https://login.microsoftonline.com/AZURE-TENANT_ID/v2.0/.well-known/openid-configuration',
+  responseMode: 'fragment',
+  extraScope: 'openid profile offline_access https://CDF-CLUSTER.cognitedata.com/user_impersonation https://CDF-CLUSTER.cognitedata.com/IDENTITY',
+  loginParams: {
+    prompt: 'select_account'
+  },
+  cluster: 'CDF-CLUSTER',
+}
+
+const audience = {
+  audience: "CUSTOM-AUDIENCE",
+};
+const AUTH0_CONFIG = {
+  openIdConfigurationUrl: "https://AUTH0-CUSTOM-DOMAIN/.well-known/openid-configuration",
+  cluster: 'CDF-CLUSTER',
+  clientId: "AUTH_0-APP-ID",
+  responseMode: "query",
+  extraScope: 'openid profile offline_access',
+  authenticateParams: audience,
+  refreshParams: audience,
+  loginParams: {
+    prompt: "login",
+  },
+};
+
+
+function App() {
+  const [token, setToken] = useState();
+  const [inspect, setInspect] = useState();
+  const [isSignedIn, setIsSignedIn] = useState(false);
+  const [error, setError] = useState(undefined);
+
+
+  const client = useMemo(() => new CogniteClient({ appId: 'sample-app-id' }), []);
+
+  useEffect(() => {
+    client.loginWithOAuth('OIDC_AUTHORIZATION_CODE_FLOW', {
```

Comment:
needs to be updated to send in flow object

Reply:
fixed